### DISCUSSION TITLE
My github have an invisible notification, which I can't mark as read

### DISCUSSION THREAD

**Discussion by JerryJian** [community/community#54596] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

My github always have a notification, which I can't mark as read. It is annoy me. 
Even I filter notifications through 'repo:codepath/android_guides', the notification not listed, so I can't operate it. 

![image](https://user-images.githubusercontent.com/800502/236591960-68c842e7-382b-49b1-8d95-e52a554fbe9e.png)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by radar-blip** [comment, marked as answer]
Its a known bug (ghost notification). See link below for a probable fix.

https://github.com/orgs/community/discussions/6874#discussioncomment-3593547

**Reply by JerryJian** [reply]
Thanks very much, finally I mark all as read following this instruction steps: https://github.com/orgs/community/discussions/6874#discussioncomment-1872442

**Reply by radar-blip** [reply]
You're welcome. Glad it worked for you.
